docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -9,57 +9,58 @@ menu:
-- [Constraint trigger is not supported](#constraint-trigger-is-not-supported)
-- [Table inheritance is not supported](#table-inheritance-is-not-supported)
-- [%Type syntax is not supported](#type-syntax-is-not-supported)
-- [GIN indexes on multiple columns are not supported](#gin-indexes-on-multiple-columns-are-not-supported)
-- [Policies on users in source require manual user creation](#policies-on-users-in-source-require-manual-user-creation)
-- [VIEW WITH CHECK OPTION is not supported](#view-with-check-option-is-not-supported)
-- [UNLOGGED table is not supported](#unlogged-table-is-not-supported)
-- [Hash-sharding with indexes on the timestamp/date columns](#hash-sharding-with-indexes-on-the-timestamp-date-columns)
-- [Exporting data with names for tables/functions/procedures using special characters/whitespaces fails](#exporting-data-with-names-for-tables-functions-procedures-using-special-characters-whitespaces-fails)
-- [Importing with case-sensitive schema names](#importing-with-case-sensitive-schema-names)
-- [Unsupported datatypes by YugabyteDB](#unsupported-datatypes-by-yugabytedb)
-- [Unsupported datatypes by Voyager during live migration](#unsupported-datatypes-by-voyager-during-live-migration)
-- [XID functions is not supported](#xid-functions-is-not-supported)
-- [REFERENCING clause for triggers](#referencing-clause-for-triggers)
-- [BEFORE ROW triggers on partitioned tables](#before-row-triggers-on-partitioned-tables)
-- [Advisory locks is not yet implemented](#advisory-locks-is-not-yet-implemented)
-- [System columns is not yet supported](#system-columns-is-not-yet-supported)
-- [XML functions is not yet supported](#xml-functions-is-not-yet-supported)
-- [Large Objects and its functions are currently not supported](#large-objects-and-its-functions-are-currently-not-supported)
-- [PostgreSQL 12 and later features](#postgresql-12-and-later-features)
-- [MERGE command](#merge-command)
-- [JSONB subscripting](#jsonb-subscripting)
-- [Events Listen / Notify](#events-listen-notify)
-- [Two-Phase Commit](#two-phase-commit)
-- [DDL operations within the Transaction](#ddl-operations-within-the-transaction)
-- [Hotspots with range-sharded timestamp/date indexes](#hotspots-with-range-sharded-timestamp-date-indexes)
-- [Redundant indexes](#redundant-indexes)
-
-### Adding primary key to a partitioned table results in an error
+Table of Content:
+
+- [Data Definition](#data-definition)
+- [Data Types](#data-types)
+- [Data Manipulation](#data-manipulation)
+- [Functions and Operators](#functions-and-operators)
+- [Indexes](#indexes)
+- [Concurrency Control](#concurrency-control)
+- [Extensions](#extensions)
+- [Server Programming](#server-programming)
+- [Postgres 12 And later features](#postgres-12-and-later-features)
+- [Migration Process and Tooling Issues](#migration-process-and-tooling-issues)
+- [Performance Optimizations](#performance-optimizations)
+
+## Data Definition
```

Comment:
Convention for headings everywhere in docs is to use sentence capitalization

Reply:
Understood!